hugo.toml

```
@@ -125,6 +125,9 @@ auroraboot_version = "v0.2.6"
 flavor = "debian"
 armFlavor = "alpine"
 registryURL = "quay.io/kairos"
+exampleImage = "opensuse"
+exampleImageCoreTag = "leap-15.5-core-amd64-generic-v2.4.1"
+exampleImageStandardTag = "leap-15.5-core-amd64-generic-v2.4.1-k3sv1.27.3-k3s1"
```

```suggestion
exampleImageCoreTag = "leap-15.5-core-amd64-generic-v2.4.1"
exampleImageStandardTag = "leap-15.5-standard-amd64-generic-v2.4.1-k3sv1.27.3-k3s1"
```

Standard one is incorrect here, but overall, I would prefer not to do this here since it will keep the versions of the examples pinned. I think it would be better to build them using the mechanism we already have with the version. 

```
"{{< registryURL >}}/{{< exampleImage >}}:leap-15.5-core-amd64-generic-{{< kairosVersion >}}"
```

Or even better, maybe introduce another function which itself does the code above and can be used like so:

```
"{{< registryURL >}}/{{< exampleStandardImage >}}
```

maybe even go as far as

```
{{< exampleStandardImage >}}
```